lib/SIPMessage.js

```diff
@@ -25,7 +25,7 @@ class OutgoingRequest
       return null;
     }
 
-    params = params || {};
+    params = Utils.cloneObject(params);
```

Comment:
this one is not needed. `params` are internally generated.

Reply:
updated